.gitmodules

```diff
@@ -1,11 +1,11 @@
 [submodule "jetstack-cert-manager"]
 	path = cert-manager
 	url = https://github.com/openshift/jetstack-cert-manager.git
-	branch = release-1.15
+	branch = release-1.16
 [submodule "cert-manager-operator"]
 	path = cert-manager-operator
 	url = https://github.com/openshift/cert-manager-operator.git
-	branch = cert-manager-1.15
+	branch = master
```

Comment:
Should this be `cert-manager-1.16`?

Reply:
We can wait for sometime till we branch out from master (second staging builds, let's say; would want to build from master initially to get signals if builds are sane), and update here after.